<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@3.8">
  <node class="cps.n.StepEndNode" plugin="workflow-cps@2.92">
    <parentIds>
      <string>9</string>
    </parentIds>
    <id>10</id>
    <startId>4</startId>
  </node>
  <actions>
    <wf.a.BodyInvocationAction plugin="workflow-api@2.46"/>
    <wf.a.TimingAction plugin="workflow-api@2.46">
      <startTime>1626421317502</startTime>
    </wf.a.TimingAction>
    <wf.a.ErrorAction plugin="workflow-api@2.46">
      <error class="java.lang.IllegalStateException">
        <detailMessage>hudson.AbortException: Fail: the returned code 401 is not in the accepted range: [[100‥399]]</detailMessage>
        <cause class="hudson.AbortException">
          <detailMessage>Fail: the returned code 401 is not in the accepted range: [[100‥399]]</detailMessage>
          <stackTrace>
            <trace>jenkins.plugins.http_request.HttpRequestExecution.responseCodeIsValid(HttpRequestExecution.java:414)</trace>
            <trace>jenkins.plugins.http_request.HttpRequestExecution.processResponse(HttpRequestExecution.java:424)</trace>
            <trace>jenkins.plugins.http_request.HttpRequestExecution.authAndRequest(HttpRequestExecution.java:329)</trace>
            <trace>jenkins.plugins.http_request.HttpRequestExecution.call(HttpRequestExecution.java:260)</trace>
            <trace>jenkins.plugins.http_request.HttpRequestExecution.call(HttpRequestExecution.java:80)</trace>
            <trace>hudson.remoting.LocalChannel.call(LocalChannel.java:46)</trace>
            <trace>jenkins.plugins.http_request.HttpRequestStep$Execution.run(HttpRequestStep.java:380)</trace>
            <trace>jenkins.plugins.http_request.HttpRequestStep$Execution.run(HttpRequestStep.java:358)</trace>
            <trace>org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution$1$1.call(AbstractSynchronousNonBlockingStepExecution.java:47)</trace>
            <trace>hudson.security.ACL.impersonate2(ACL.java:449)</trace>
            <trace>hudson.security.ACL.impersonate(ACL.java:461)</trace>
            <trace>org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution$1.run(AbstractSynchronousNonBlockingStepExecution.java:44)</trace>
            <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
            <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
            <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)</trace>
            <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)</trace>
            <trace>java.lang.Thread.run(Thread.java:748)</trace>
          </stackTrace>
          <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" resolves-to="java.util.Collections$UnmodifiableList">
            <c class="list"/>
            <list reference="../c"/>
          </suppressedExceptions>
        </cause>
        <stackTrace>
          <trace>jenkins.plugins.http_request.HttpRequestExecution.call(HttpRequestExecution.java:263)</trace>
          <trace>jenkins.plugins.http_request.HttpRequestExecution.call(HttpRequestExecution.java:80)</trace>
          <trace>hudson.remoting.LocalChannel.call(LocalChannel.java:46)</trace>
          <trace>jenkins.plugins.http_request.HttpRequestStep$Execution.run(HttpRequestStep.java:380)</trace>
          <trace>jenkins.plugins.http_request.HttpRequestStep$Execution.run(HttpRequestStep.java:358)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution$1$1.call(AbstractSynchronousNonBlockingStepExecution.java:47)</trace>
          <trace>hudson.security.ACL.impersonate2(ACL.java:449)</trace>
          <trace>hudson.security.ACL.impersonate(ACL.java:461)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.AbstractSynchronousNonBlockingStepExecution$1.run(AbstractSynchronousNonBlockingStepExecution.java:44)</trace>
          <trace>java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)</trace>
          <trace>java.util.concurrent.FutureTask.run(FutureTask.java:266)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)</trace>
          <trace>java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)</trace>
          <trace>java.lang.Thread.run(Thread.java:748)</trace>
        </stackTrace>
        <suppressedExceptions class="java.util.Collections$UnmodifiableRandomAccessList" reference="../cause/suppressedExceptions"/>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>